coin_prices_from_coingecko/README.md

```diff
@@ -86,7 +86,7 @@ using the `bky-as` CLI by passing in the
     "coin_id": "bitcoin"
   },
   "secret": {
-    "api_key": "CoinGecko API key"
+    "api_key": "{{.YOUR_COINGECKO_API_KEY}}"
```

Comment:
README updated to reflect the small changes I made in the coingecko example.

Reply:
Thanks for making sure to update the readme. Could you also update the paragraph below to explain what the user needs to update (everything between the quotes) and why we are using this syntax in the config file. The explanation doesn't need to be super detailed.

Reply:
yep, can do
- [ ] add explanation of template formatting

Reply:
done